test/integration/test.js

```diff
@@ -94,6 +96,20 @@ describe('svg-react-loader', () => {
             );
     });
 
+    it('styles.svg - with interpolation', () => {
+        const wrapper = mount(<StylesInterpolation />);
+
+        if (!wrapper.html().match(/\.my-prefix-styles-1f3e1-foo/)) {
```

Comment:
I think these are fine.

I would also like to see the positive assertions (that the final class name is in the correct format.